src/docs/build-deploy-and-maintain-apps/maintain-an-application.md

```diff
@@ -0,0 +1,498 @@
+
+3. It's essential for your team to have a strong understanding of your application and its relevant components.  This knowledge base is crucial for effective troubleshooting and minimizing the impact of any downtime
+
+---
+## Maintain security 
+
+The OpenShift platform boasts numerous features that enhance security compared to many traditional hosting environments. Nonetheless, certain areas demand additional focus to guarantee the security of your application.  These include:
+* [Network Policies](#network-policies)
+* [RoleBindings](#rolebindings)
+* [Vault](#Vault)
+
+### Network Policies
+Upon creation, your namespaces are equipped with a **"deny all"** NetworkPolicy called 'platform-services-controlled-default' by default. 
+
+ This policy effectively restricts all traffic, including within your own namespace, from reaching your services.   
+ 
+ This default configuration prioritizes security. To make your application functional, you'll need to establish additional network policies.
+
+* When creating network policies, **do not create an "allow all" policy.**
+* Allow only specific traffic that is needed.
+* For more information, see [OpenShift network policies](https://docs.developer.gov.bc.ca/openshift-network-policies/)
+
+Review your network policies:
+
+* Are they overly permissive?  
+* Do you allow inbound traffic from the Internet to services other than those that are meant for public access?
+
+Keep in mind that there's a production cluster called Emerald, which employs the NSX software-defined network (SDN) unlike the default OpenShift SDN used by other clusters. This choice enables more precise configuration, including thorough egress and ingress controls. 
+
+If your application has advanced network security needs, consider utilizing the Emerald cluster. If you have any questions regarding this, don't hesitate to contact the Platform Services team.
+
+### RoleBindings
+Similar to network policies, keep your RoleBindings within namespaces as streamlined as possible. By default, anyone linked to the product in the platform's Registry will have administrative access to all four namespaces. These namespace administrators can generate additional RoleBindings as required to facilitate your application.
+
+* Only grant namespace access to those who need it
+* Grant minimal permissions.  How much access do your users need?
+    * There is a 'view' ClusterRole for read-only access
+    * For special cases, like ServiceAccounts, create a Role and RoleBinding to grant it only the permissions that it needs to fulfill its function
+* Create separate RoleBindings for users with differing access needs.  That is, if 'User A' requires edit access and 'User B' requires only view access, create separate RoleBindings rather than give edit access to both.
+* Periodically review the RoleBindings in your namespaces.  As your team changes, remove access for those that leave.
```

Comment:
```suggestion
* Periodically review the RoleBindings in your namespaces.  As your team changes, remove access for those that leave.
* Store your Roles and RoleBindings as YAML along with your other objects in your source code repo so that they can be easily re-created
```